lms/views/dashboard/api/assignment.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, request) -> None:
     @view_config(
         route_name="api.dashboard.assignments",
         request_method="GET",
-        renderer="json",
+        renderer="json_iso_utc",
```

Comment:
Using it first on the assignment API to first test the approach here. As I mention on the description ideally we'll add this to to all dashboards first and the case by case to all other endpoints that now use `json`.

If we go with this approach I'll make a note in the issue to track progress of the switch.